client/app/components/modal/CreateEventModal.jsx

```diff
@@ -126,7 +126,8 @@ export default class CreateEventModal extends React.Component {
         <ModalFooter
           closeModal={this.props.closeModal}
           saveChanges={this.saveChanges}
-          disableSaving={this._savingDisabled()} />
+          unresolvedErrors={!isEmpty(this.state.errors)}
```

Comment:
Not sure whether `unresolvedErrors` is the best name here. We keep true/false flag inside it right? The name of variable suggests that it contains collection of `unresolvedErrors`. Maybe `hasUnresolvedErrors` ?